src/commands/publish/preview/upload.rs

```diff
@@ -45,13 +47,24 @@ pub fn upload_and_get_id(
             if missing_fields.is_empty() {
                 let client = http::auth_client(&user);
 
+                // todo add sites to bucket here
```

Comment:
what exactly do you mean here? is there an issue filed for this?

Reply:
This is for live preview for sites :) #653